### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by takana22**

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -16,9 +16,10 @@
     </ul>
     <% if user_signed_in? %>
       <p>You are signed in as <%= current_user.email %></p>
+      <%= link_to "log out",  destroy_user_session_path, method: :delete %>
     <% else %>
-      <a class="nav-link" href="#">Login</a>
-      <a class="nav-link" href="#">Sign up</a>
+      <%= link_to "login in", new_user_session_path, class: "nav-link" %>
   <% end %>
+  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
```
Comment (Laklemme) [inline app/views/shared/_navbar.html.erb:23]: good job! could that go into the if statement as well?

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-  <div class="form-actions">
-    <%= f.button :submit, "Log in" %>
-  </div>
 <% end %>
-
-<%= render "devise/shared/links" %>
+    </div>
+  </div>
+</div>
```
Comment (Laklemme) [inline app/views/devise/sessions/new.html.erb:20]: Looks amazing!

File: app/views/dinos/new.html.erb [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+       </div>
+     <div class="form-actions">
+    <%= f.button :submit, "Log in", class: "btn btn-primary" %>
+  </div>
+  <%= render "devise/shared/links" %>
+  <%= render 'form', dino: @dino %>
+<% end %>
+    </div>
+  </div>
+</div>
```
Comment (Laklemme) [inline app/views/dinos/new.html.erb:21]: Why is the on the dinos new page?

**Review by lmllr — COMMENTED** [review]
<3

**Review by lmllr — APPROVED** [review]
and go

**Pull request merged**